Userland/Libraries/LibGfx/ImageFormats/PNGLoader.cpp

```diff
@@ -196,10 +198,40 @@ ErrorOr<bool> PNGImageDecoderPlugin::initialize()
         m_context->exif_metadata = TRY(TIFFImageDecoderPlugin::read_exif_metadata({ exif_data, exif_length }));
     }
 
+    if (m_context->exif_metadata) {
+        if (auto result = m_context->apply_exif_orientation(); result.is_error())
+            dbgln("Could not apply eXIf chunk orientation for PNG: {}", result.error());
```

Comment:
EXIF?

Reply:
That's already merged, but you can always leave a separate PR to fix typos

Reply:
It's not a typo. The name of the PNG chunk is eXIf: http://ftp-osl.osuosl.org/pub/libpng/documents/pngext-1.5.0.html#C.eXIf